Jenkinsfile

```diff
@@ -0,0 +1,52 @@
+pipeline {
+    agent {
+        label 'Performance'
+    }
+    tools {
+        maven "Apache Maven 3.6"
+        jdk "OpenJDK 11 Latest"
+    }
+    stages {
+        stage('Initialize') {
```

Comment:
Seems like you mixed spaces and tabs in this file, resulting in weird indentation.